app/view_objects/forums/view.rb

```diff
@@ -81,7 +81,7 @@ def user_subscriptions
   end
 
   def forum_channel forum_id
-    "/forum-#{forum_id}/#{h.locale_from_host}"
+    "/forum-#{forum_id}/ru"
```

Comment:
@Yokitori давай до конца удалим локаль из названия канала faye
```suggestion
    "/forum-#{forum_id}"
```